src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -241,6 +243,43 @@ export function transform(formConfig, form) {
         )
       : formData;
 
+  // new disabilities that match a name on our mapped list need their
+  // respective classification code added
+  const addClassificationCodeToNewDisabilities = formData => {
+    const { newDisabilities } = formData;
+    if (!newDisabilities) {
+      return formData;
+    }
+
+    const flippedDisabilityLabels = {};
+    Object.entries(disabilityLabels).forEach(([code, description]) => {
+      flippedDisabilityLabels[description.toLowerCase()] = code;
+    });
+
+    const newDisabilitiesWithClassificationCodes = newDisabilities.map(
+      disability => {
+        const { condition } = disability;
+        if (!condition) {
+          return disability;
+        }
+        const loweredDisabilityName = condition.toLowerCase();
```

Comment:
What do you think about standardizing on a `normalizeDisabilityName` or something? Even if for now that is just
```js
const normalizeDisabilityName = name => name.toLowerCase();
```